File: Composite Pattern/PHP/City.php
<?php

require_once "Person.php";
require_once "Man.php";
require_once "Women.php";


class City 
{
    protected static $people = [];

    public static function addCitizen(Person $person)
    {
        self::$people[] = $person;
    }

    public static function showInfo()
    {
        foreach(self::$people as $person) {
            echo $person->getInfo() . PHP_EOL;
        }
    }
}

?>

File: Composite Pattern/CS/City.cs
using System;
using System.Collections.Generic;

namespace CompositePattern
{
    public class City
    {
        private readonly List<Person> _people;

        public City()
        {
            this._people = new List<Person>();
        }

        public void AddCitizen(Person person)
        {
            this._people.Add(person);
        }

        public void GetFull()
        {
            foreach (var person in this._people)
            {
                Console.WriteLine(person.GetInfo() + ":" + person.GetName() + " | " + person.GetAge());
            }
        }
    }
}